easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-goolf-1.7.20.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
```

Comment:
I suggest to add an extra Authors:: line with your name, if you prefer.
This permits to address a bunch of interested/related people easily (git shows only committers btw).